Source/Client.swift

```diff
@@ -106,6 +107,7 @@ extension Client {
             )
 
             try connection.open(timingOut: now() + connectionTimeout)
```

Comment:
Actually no, it's fine the way it is, this is in the branch where the connection is getting created new (look at the full changes)